ui/desktop/src/components/ConfigContext.tsx

```diff
@@ -122,6 +122,8 @@ export const ConfigProvider: React.FC<ConfigProviderProps> = ({ children }) => {
         body: query,
       });
       await reloadConfig();
+      // Force refresh extensions list to ensure UI is updated
+      setExtensionsList([]);
```

Comment:
I'm a little confused on the need for this one. Doesn't:

```await getExtensions(true)``` in fetch extensions already do a force reload?

The logic for loading extensions is very fiddly and that's my fault. I just want to be sure of the need for more imperative things to get extensions to reload before we merge.